package.json

```diff
@@ -3,10 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@glennsl/bs-json": "3.0.0",
-    "@rescript/react": "^0.10.3",
-    "bs-fetch": "^0.3.0",
-    "bs-json": "^1.0.1",
+    "@glennsl/rescript-fetch": "^0.2.0",
```

Comment:
I still kept rescript-fetch for simplicity reasons. Usually I don't like to rely on libraries like this bc the API surface is hard to navigate, and a simple on-demand binding to the global `fetch` api is more straight-forward.

We don't use fetch too heavily so this should be fine.